themes/startwords/layouts/shortcodes/deepzoom.html

```diff
@@ -3,14 +3,16 @@
 {{- $height := default "30em" (.Get "height") -}}
 <div class="deepzoom" id="{{ $zoom_id }}" style="height: {{ $height }}"></div>
 <script>
-OpenSeadragon({
-    id: "{{ $zoom_id }}",
-    prefixUrl: "https://cdn.jsdelivr.net/npm/openseadragon@2.4/build/openseadragon/images/",
-    preserveViewport: true,
-    visibilityRatio:    1,
-    minZoomLevel:       1,
-    defaultZoomLevel:   1,
-    gestureSettingsMouse: { scrollToZoom: false },
-    tileSources: "{{ if .IsNamedParams }}{{ .Get "tile" }}{{ else }}{{ .Get 0 }}{{ end }}",
-});
+    window.addEventListener("onload", function() {
```

Comment:
Whoops, I should have done it that way when I implemented it!

Reply:
OpenSeadragon isn't actually a very heavy lift; it's mainly the Sketchfab embed that is a huge resource (I think it's an entire React app with Redux!). I've tried some things to mitigate it but the lighthouse scores are still pretty bad on pages that include it, so I need to investigate more.